src/ace.tsx

```diff
@@ -342,7 +342,7 @@ export default class ReactAce extends React.Component<IAceEditorProps> {
     // First process editor value, as it may create a new session (see issue #300)
     if (
       this.editor &&
-      nextProps.value &&
+      nextProps.value !== undefined &&
```

Comment:
I think it should be `nextProps.value != null`, this will check against both `null` and `undefined`

Reply:
Why does null to be included into the check?

Reply:
It's best practice to check for both `null` and `undefined`. We just want to fix the bug when an empty string is set, not when null value is set